core/src/test/java/org/apache/calcite/test/SqlJsonFunctionsTest.java

```diff
@@ -63,6 +64,7 @@
   public static final String INVOC_DESC_JSON_VALUE_ANY = "jsonValueAny";
   public static final String INVOC_DESC_JSON_QUERY = "jsonQuery";
   public static final String INVOC_DESC_JSONIZE = "jsonize";
+  public static final String INVOC_DESC_PRETTY_JSONISE = "prettyJson";
```

Comment:
I would suggest to change the name `PRETTY_JSONISE` to `JSON_PRETTY`, also the string value and test method names below. Using a unified name convention will benefit new developers when they do a code searching.

Reply:
Since the methods are defined in `enum BuiltInMethod`, could you eliminate these INVOC_ constant strings, and use `BuiltInMethod` constants instead? The advantage of `BuiltInMethod` is that it validates whether each of the methods actually exist when the class is loaded.